src/main/java/uk/gov/justice/hmpps/prison/api/model/Account.java

```diff
@@ -39,6 +42,11 @@
     @NotNull
     private BigDecimal savings;
 
+    @ApiModelProperty(required = true, value = "Damage obligation balance.")
+    @NotNull
+    @JsonDeserialize(using = MoneyDeserializer.class)
```

Comment:
should we make sure the other amounts are in the correct format?

Reply:
[example](blob:https://dsdmoj.atlassian.net/e872b7af-ec6d-4023-90f9-c128a85c11ee#media-blob-url=true&id=e3944a83-063c-429b-a5cb-4007fe08b737&collection=&contextId=76287&mimeType=image%2Fpng&name=Screenshot%202020-11-20%20at%2014.39.38.png&size=74001&width=1929&height=137)

Reply:
done